python/tests/test_table_read.py

```diff
@@ -193,16 +193,6 @@ def test_read_table_with_stats():
     # assert data.num_rows == 0
 
 
-def test_read_table_with_only_struct_stats():
```

Comment:
I'm removing this because everything should now be covered by the lower level rust test I added. This did surface something interesting though. It seems like the `DeltaTable.to_pyarrow_schema()` function cannot handle the struct of array of map type I put in my test data. I think this is probably an issue for another time though.